apps/learner/src/routes/(app)/+page.svelte

```diff
@@ -50,29 +50,25 @@
 </div>
 
 <div class="z-100 pointer-events-none fixed inset-x-0 bottom-0">
-  <div class="mx-auto max-w-5xl px-4">
+  <div class="mx-auto max-w-5xl px-4 py-3">
     <div class="flex justify-end gap-x-4">
       {#if isFloatingPlayerVisible}
-        <div class="pointer-events-auto flex-grow overflow-x-hidden py-3">
-          <FloatingPlayer
-            title="Navigating Special Educational Needs in Singapore: A Path to Inclusion"
-            isplaying={audioState.isPlaying}
-            onplay={togglePlayPause}
-            onclick={handleFloatingPlayerClick}
-          />
-        </div>
+        <FloatingPlayer
+          title="Navigating Special Educational Needs in Singapore: A Path to Inclusion"
+          isplaying={audioState.isPlaying}
+          onplay={togglePlayPause}
+          onclick={handleFloatingPlayerClick}
+        />
       {/if}
-      <div class="pointer-events-auto py-3">
-        <FloatingChat />
-      </div>
+
+      <FloatingChat />
     </div>
   </div>
 </div>
 
 {#if isModalOpen}
   <div class="z-200 fixed inset-0 bg-slate-950">
     <div class="mx-auto w-full max-w-5xl px-4 py-3">
-      <!-- Modal Header -->
```

Comment:
Should keep this comment so it's easier for us to segregate and reference the divs.